In [3]:
import pandas as pd

path = "C:\PERKULIHAAN\SEMESTER 5\CERTAN\CERTAN\week 2\shopping_trends.csv"

df = pd.read_csv(path)

previous_purchases = df["Previous Purchases"]

print("=== Statistik Previous Purchases ===")
print("Mean :", previous_purchases.mean())
print("Std  :", previous_purchases.std())
print("Min  :", previous_purchases.min())
print("Q1   :", previous_purchases.quantile(0.25))
print("Q2   :", previous_purchases.quantile(0.50))
print("Q3   :", previous_purchases.quantile(0.75))
print("Max  :", previous_purchases.max())

=== Statistik Previous Purchases ===
Mean : 25.35153846153846
Std  : 14.447125170462309
Min  : 1
Q1   : 13.0
Q2   : 25.0
Q3   : 38.0
Max  : 50


In [2]:
df[['Age', 'Purchase Amount (USD)']].corr()

,Age,Purchase Amount (USD)
Age,1.000000,-0.010424
Purchase Amount (USD),-0.010424,1.000000


In [3]:
df[['Purchase Amount (USD)', 'Review Rating']].corr()

,Purchase Amount (USD),Review Rating
Purchase Amount (USD),1.000000,0.030776
Review Rating,0.030776,1.000000


# Tugas 2
## Analisis Dataset Customer Shopping Trends

### Tujuan
Melakukan analisis statistik deskriptif dan korelasi pada dataset Customer Shopping Trends.

### A. Statistik Deskriptif Previous Purchases

Variabel yang digunakan adalah:

`Previous Purchases`

Berdasarkan analisis statistik deskriptif, nilai rata-rata Previous Purchases adalah 25.351538, dengan nilai minimum 1 dan maksimum 50.

Hasil korelasi menunjukkan bahwa hubungan Age dengan Purchase Amount (USD) sangat lemah dengan nilai -0.010424. Sementara itu, hubungan Purchase Amount (USD) dengan Review Rating juga sangat lemah dengan nilai 0.030776.
